egs/audioset/AT/local/generate_audioset_manifest.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import csv
+
+import torch
+import torchaudio
+import logging
+import glob
+from lhotse import load_manifest, CutSet, Fbank, FbankConfig, LilcomChunkyWriter
+from lhotse.cut import MonoCut
+from lhotse.audio import Recording
+from lhotse.supervision import SupervisionSegment
+from argparse import ArgumentParser
```

Comment:
please sort the import with `isort`, which follows https://peps.python.org/pep-0008/#imports

```
Imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.
```

Reply:
Fixed